ncbi/pom.xml

```diff
@@ -0,0 +1,208 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.reactome.release</groupId>
+	<artifactId>ncbi</artifactId>
+	<packaging>jar</packaging>
+	<version>1.0-SNAPSHOT</version>
+	<name>ncbi</name>
+	<url>http://maven.apache.org</url>
```

Comment:
This element should be used to refer to the URL for the project that is built by this pom.

Reply:
Removed!